spec/board_spec.rb

```diff
@@ -8,13 +8,24 @@
       board = Board.new
       expect(board).to be_an_instance_of(Board)
     end
+
+    it "prints a welcome message" do
+      board = Board.new
+      def print_welcome
```

Comment:
I would have never thought of this to bypass the confusion of using RSpec to test output. Nice workaround!